src/viking.js

```diff
@@ -1,11 +1,47 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage) {
+        this.health -= damage
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength){
+        super(name, health, strength);
+        this.name = name;
+        this.health = health;
+        this.strength = strength;
```

Comment:
`super(health, strength);` en vez de `super(name, health, strength);`
=> no hace falta volver a declarar this.health y this.strength ya que Viking los hereda de Soldier con `super(health, strength) `